### ISSUE TITLE
Profile version representation by identifier

### ISSUE THREAD

**Issue by nicholascar** [w3c/dxwg#391]
if we use URIs for profiles but the profile definition changes, how will clients know? Do we need to make strong statements about versions and URI uniqueness?

**Comment by nicholascar** [comment]
Should this issue be pushed to the Guidance group?

**Comment by aisaac** [comment]
@nicholascar yes and I believe it could be relevant for the UCR group as well. And maybe the entire DXWG - I suspect DCAT people may have something to say too ;-)

**Comment by kamhayfung** [comment]
Just a thought. The version ID of a profile could be added to its URI for precision. If the ID is absent, the URI may refer to the latest version of the profile.

**Comment by larsgsvensson** [comment]
@kamhayfung scripsit:
> Just a thought. The version ID of a profile could be added to its URI for precision. If the ID is absent, the URI may refer to the latest version of the profile.

That would require that we mandate some kind of URI pattern so that agents would know if the URI is versioned or not. In practice I guess most organisations will offer this one way or the other but I don't think we can specify how that will be done. Perhaps we can suggest a pattern for linking a versioned URI to a "latest version" one (`dct:isversionOf`?)

**Comment by kamhayfung** [comment]
> 
> 
> @kamhayfung scripsit:
> 
> > Just a thought. The version ID of a profile could be added to its URI for precision. If the ID is absent, the URI may refer to the latest version of the profile.
> 
> That would require that we mandate some kind of URI pattern so that agents would know if the URI is versioned or not. In practice I guess most organisations will offer this one way or the other but I don't think we can specify how that will be done. Perhaps we can suggest a pattern for linking a versioned URI to a "latest version" one (`dct:isversionOf`?)

It depends of how far this document covers but some non-normative suggestions would definitely be beneficially to adopters. Both Skos and Dublin core offer some kinds of versioning support and they can be considered. 

If the version ID is not in the URI, the ID can be derived from the profile definition by first obtaining a list of profile URLs and then downloading their profile definitions (incl. a mix of dct:isVersionOf, dct:hasVersion, dct:replaces) from a respective source. However, this approach is human readable but not very human friendly.
